components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3980,7 +3980,10 @@ def definition_body(self):
 let args = CallArgs::from_vp(vp, argc);
 let global = GlobalScope::from_object(args.callee());
 """
-        if len(self.exposureSet) == 1:
+
+        if len(self.exposureSet) == 2 and "Window" in self.exposureSet:
+            preamble += "let global = DomRoot::downcast::<dom::types::Window>(global).unwrap();\n"
```

Comment:
This and the below checks are for exposureSet, which contains the exact `["DissimimalrOriginWindow" "Window"]`.
Before this update, they are just `["Window"]`. It seems `DissimimalrOriginWindow` is for internal usage. So in this case it will always be "Window". If you want, I can add more checks to make sure the set is exactly `["DissimimalrOriginWindow" "Window"]`